src/main.rs

```diff
@@ -90,7 +90,8 @@ async fn main() -> Result<()> {
         settings.config.num_of_thread,
         settings.config.max_sub_compactions,
     );
-    if args.repair {
+
+    if is_local_config && args.repair {
```

Comment:
I think this if conditional can be simplified for humans like

```
if args.repair {
  if !is_local_config {
   // bail
  } else {
    // do repair
  }
}
```

This structure might make it easier to follow the logic at a glance. Only if you agree, I'd like to suggest changing like above.

Reply:
I think Clippy would say that what @sophie-cluml suggested doesn’t follow one of its lints.

Reply:
@sehkone My suggestion does not produce Clippy warning.